libs/datamodel/parser-database/src/types.rs

```diff
@@ -98,7 +99,7 @@ impl ScalarFieldType {
     }
 }
 
-#[derive(Debug)]
+#[derive(Debug, Clone)]
```

Comment:
(update: haven't found it, but probably just didn't see)

Reply:
Needed because it's now used in `CompositeTypeField` which requires `Clone` https://github.com/prisma/prisma-engines/blob/b61b8a89e460e7230a8d6f8b4865386eaa743d37/libs/datamodel/parser-database/src/types.rs#L68

Reply:
Hmm none of these things really require clone, so far, is it for the walker? If so let's just merge, I'll move that to the better approach (the generic struct) later.

Reply:
And `CompositeTypeField` requires `Clone` because `ScalarFieldType` does.

Reply:
I'll do a follow-up PR removing all the Clone impls, no need to worry on this one.